src/controls.actions.js

```diff
@@ -680,9 +680,14 @@
     var target = transform.target, localPoint = getLocalPoint(transform, transform.originX, transform.originY, x, y),
         strokePadding = target.strokeWidth / (target.strokeUniform ? target.scaleX : 1),
         multiplier = isTransformCentered(transform) ? 2 : 1,
+        oldWidth = target.width, hasScaled,
         newWidth = Math.abs(localPoint.x * multiplier / target.scaleX) - strokePadding;
     target.set('width', Math.max(newWidth, 0));
-    return true;
+    hasScaled = oldWidth !== newWidth;
+    if (hasScaled) {
+      fireEvent('scaling', commonEventInfo(eventData, transform, x, y));
+    }
+    return hasScaled;
```

Comment:
i would swear it was firing `resizing` did i dream of it?

Reply:
Must have been some conversation with some developer, but i would prefer not to fire scaling since scale is actually not change, can we call the event something else and you subscribe to both? we also may have extra events somewhere after an action is performed. does `resizing` sounds good?

Reply:
Sure, either way works for me. If we do change it to resizing we should list it in the breaking changes doc.